### ISSUE TITLE
Throw Error: configmap "dapr-ambient-configmap" not found

### ISSUE THREAD

**Issue by thangchung** [dapr-sandbox/dapr-shared#26]
I followed the guidance at https://github.com/dapr-sandbox/dapr-ambient/blob/8ed4f260f44249aba275a54404506f3080deed61/tutorial/README.md

The only difference is I changed the version of `daprd` as

```sh
helm install product-api-dapr-ambient ambient/dapr-ambient --set ambient.appId=product-api --set ambient.remoteURL=product-api --set daprd.image.tag=1.11.0-rc.7
```

But when I run the `helm install` above, it always threw the exception 

```sh
15s         Warning   Failed                    pod/product-api-dapr-ambient-mgspf                     Error: configmap "dapr-ambient-configmap" not found
```

I checked all the helm charts in this repo, but couldn't find any clue to fix it.

Could anyone show me how can I overcome this issue? Thank you very much.

**Comment by salaboy** [comment]
@thangchung hi, let's fix this issue.. 
We are currently working on producing the final artifacts to be hosted in the official repos, so my guess is that you are getting an old version of the chart and init container.

**Comment by thangchung** [comment]
Thanks, @salaboy for the answer. Any way to overcome it now? Or do I need to wait for the fix?

**Comment by salaboy** [comment]
can you check which init-container are you using? If you describe the pod that ambient deploys you will see that an init container runs.

**Comment by salaboy** [comment]
@thangchung you can override that init container here: https://github.com/dapr-sandbox/dapr-ambient/blob/main/chart/dapr-ambient/templates/_daemondeployshared.yaml#L23 

And build the latest version from source by running: 
```
make init/build-image
```

That will create a docker container that you can push to docker hub and then use that to replace the values in the chart. 
We are trying to automate all this for you so you can consume the new container with the latest changes and the helm chart from the official Dapr repositories.

**Comment by salaboy** [comment]
fixed in v0.0.10
